admin/services/pin.service.js

```diff
@@ -52,11 +52,15 @@ pinService.expirePupilPin = async (token, checkCode) => {
 /**
  * Check and expire pupil pins based on set of provided pupil Ids
  * @param pupilIds
+ * @param {Number} schoolId - `school.id` database ID
  * @returns {Promise}
  */
 
-pinService.expireMultiplePins = async (pupilIds) => {
-  const pupils = await pupilDataService.sqlFindByIds(pupilIds)
+pinService.expireMultiplePins = async (pupilIds, schoolId) => {
```

Comment:
@jon-shipley Do you think it would also makes sense here to check for the pupilIds existence?

Reply:
Possibly, but it's not required as the underlying data-access method will throw if they are not there.